test/tools/libtesteth/ImportTest.cpp

```diff
@@ -855,11 +896,11 @@ void ImportTest::traceStateDiff()
 
     for(auto const& tr : m_transactions)
     {
-        if (network == netIdToString(tr.netId) || network == "ALL")
+        if (network == tr.netId || network == "ALL")
```

Comment:
(Nit) Perhaps we should make this comparison case-insensitive?

Reply:
It's case-sensitive everywhere in testeth, I'd leave it like this.

(these strings are actually field names test JSON files, so proper behaviour depends on whether field names are case-sensitive in JSON, are they?)

Reply:
Makes sense, if it's already case sensitive let's leave it alone.